portfolio/src/main/webapp/index.html

```diff
@@ -73,6 +67,25 @@ <h1 id="greeting-container"></h1>
           </a>
         </div>
       </div>
+      <br>
+      <div class="row text-center">
+        <div class="col">
+          <label for="comment-text">Comment:</label><br>
+          <textarea id="comment-text" name="comment-text" form="comment-form" required></textarea>
+          <form action="/data" method="POST" id="comment-form">
+            <label for="comment-author">Your Name <br> (Leave blank to publish anonymously):</label><br>
+            <input type="text" id="comment-author" name="comment-author">
+            <br><br>
+            <input type="submit" />
+          </form>
+        </div>
+      </div>
+      <br>
+      <div class="row">
+        <div class="col-md-6">
```

Comment:
Nit: I haven't tried this myself, but you might consider combining the "col-md-6" and "comments-container" divs into something like `<div id="comments-container" class="col-md-6">`.  If this works (without changing the layout) it will make the HTML a bit more readable.

Reply:
Sounds good! I just tried this, and it works.